packages/wallet/src/__stories__/index.tsx

```diff
@@ -16,27 +17,28 @@ const walletStateRender = state => () => {
   );
 };
 
-export const protocolStateRender = (protocolState: ProtocolState) => {
-  const walletState = {
-    ...initializedState,
-    processStore: {
-      dummyProcessId: {
-        processId: 'dummyProcessId',
-        protocol: 0, // at the moment this is not used by containers
-        protocolState: nestProtocolState(protocolState),
-        channelsToMonitor: [],
-      },
-    },
-    currentProcessId: 'dummyProcessId',
-  };
-
-  return walletStateRender(walletState);
+const protocolStateRender = (protocolState: ProtocolState, Container) => () => {
+  // TODO type Container
+  return (
+    <Provider store={fakeStore(protocolState)}>
+      <Modal
+        isOpen={true}
+        className={'wallet-content-center'}
+        overlayClassName={'wallet-overlay-center'}
+        ariaHideApp={false}
+      >
+        <StatusBarLayout>
+          <Container state={protocolState} />
```

Comment:
Does this work for containers that are not "top-level"? In other words, what if a container is passed that the wallet displays inside another container?

Reply:
That's not a problem currently, since all the containers do as a whole is to consume a top level protocol state, drill down to the lowest level protocol state that is nested, and then return the appropriate component. In the wild, a non-top-level container is going to return this component to its parent container, and on to its parent (etc). Here we just shortcut that process and get the component directly from its neatest container. It can then be wrapped in a bit more frontend code and rendered.